test/core/exceptions.wast

```diff
@@ -0,0 +1,11 @@
+;; Test exception handling support
```

Comment:
Nit: Maybe name this file `event.wast`, since it's testing event definitions (exception-related instructions should go to separate files).

Reply:
Done